c8/c8ql.py

```diff
@@ -194,6 +196,8 @@ def execute(self,
         :param stream: Specify *true* and the query runs as a **stream**.
             The query result is not stored on server, but calculated on the fly.
         :type stream: bool
+        :param stream: Specify *true* and write sql query.
```

Comment:
param name is "sql" here I guess, not stream

Reply:
👍